docs/variants/pc_engines/firmware-update.md

```diff
@@ -31,12 +31,12 @@ protect pin jumper on J2 (apu2) or J3 (apu3/4/6).
 
     The settings of all the above options can be restored after a firmware update.
 
-=== "(coreboot+SeaBIOS) firmware"
+    ## Updating Dasharo
 
-    No additional steps required.
+    ```bash
+    flashrom -p internal -w pcengines_apu<variant>_v<version>.rom --fmap -i WP_RO -i RW_SECTION_A
+    ```
 
-## Updating Dasharo
+=== "(coreboot+SeaBIOS) firmware"
 
-```bash
-flashrom -p internal -w pcengines_apu<variant>_v<version>.rom --fmap -i WP_RO -i RW_SECTION_A
-```
+    1. Power on the device.
```

Comment:
Power on or reboot?

Reply:
@miczyg1, why reboot? I do not consider this section finished. An update should be done from DTS, but it does not support Dasharo (coreboot+SeaBIOS) yet, so this section is incomplete. In theory, I could place here just a flashrom command to be run from network/USB-booted DTS, but I would like to fix that in roughly the correct way: https://github.com/Dasharo/meta-dts/pull/135

Reply:
But you already have the flashrom command above. So I thought these are the steps after executing flashrom.

Reply:
> But you already have the flashrom command above. So I thought these are the steps after executing flashrom.

Please note that the above flashrom command belong to coreboot+UEFI section

> I find it particularly hard to read what is on your mind. You say this section is incomplete, yet the PR is not even marked as WIP/Draft. I'm kind of lost on how I should look at such PRs.

Yes, I should mark it as WIP. I'm looking for suggestions on what is missing in this documentation, except this section, for which improvements are blocked by a lack of DTS support.

Reply:
> Please note that the above flashrom command belong to coreboot+UEFI section

Ahh right, it was hidden in the preview block (as that part was not modified). Sorry

Reply:
I assume it can be resolved.